app/views/completed_transaction/_register_to_vote_promotion.html.erb

```diff
@@ -0,0 +1,17 @@
+<div id="register-to-vote-promotion"
+     data-module="auto-track-event"
+     data-track-category="linkTrack"
+     data-track-action="linkDisplayed"
+     data-track-label="Register">
+  <p>You must <a href="/register-to-vote">register to vote</a> by 7 June if you want to take part in the EU referendum. You can register online and it only takes 5 minutes.</p>
```

Comment:
Oh, hmm, good point.  At the time this was originally written that was correct, but clearly not now.   Maybe this whole commit can go away as I can't see where these promos partials are called anymore.